fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/dimension/GenericDimensionConfigs.java

```diff
@@ -59,7 +59,7 @@ public GenericDimensionConfigs(Supplier<List<? extends DataSourceConfiguration>>
     }
 
     /**
-     * Get all dimension configurations.
+     * Get all dimension configurations of all data sources.
```

Comment:
This is added to annotate the difference between this method and `getDimensionConfigs(String dataSourceName)` which returns all dimension configurations of a **particular** data source, not all